### DISCUSSION TITLE
Feature request: conditional From Field visiblity

### DISCUSSION THREAD

**Discussion by jonas-grgt** [charmbracelet/huh#589] [Ideas]
Forms with conditional field visibility are not currently supported. 

I want the fields visibility and participation in a From Group to depend on the value of another Field.

One idea I had to add this is using the same pattern used for other dynamic features of forms.

```go
huh.NewForm(huh.NewGroup(huh.NewInput().EnableFunc(func() bool() { ...
```

**Comment by bashbunni** [comment]
Hey @jonas-grgt , thank you so much for wanting to make this project better! I've converted this to a discussion so we can talk through it and keep our backlog clear. If it does end up being something we want to add, we'll promote it back to an issue when we are ready to work on it.

This feels like something that we should be able to accomplish with Huh's dynamic functionality 🤔 ... It seems that the current state of Huh allows us to hide groups, but not fields. I would want to see if there's a way to do this as-is and compare that to a potential solution 💭
